docs/contributing_guide.md

```diff
@@ -55,6 +55,8 @@ All pull requests will be reviewed by the JPO-ODE team. The team member will eit
 ### Quarterly Release
 At the end of each quarter, a 'release/version' branch will be created from the develop branch for all release changes. This branch should be used to stabilize the code and prepare for a release. Any non-release changes continue to be pushed to the develop branch as part of the ongoing development process. Validation testing will be conducted on the release branch to ensure that the code is stable and ready for release. Once the release branch is stable, it will be merged into the master branch and tagged with the release version. If hotfixes are necessary during validation, a hotfix branch is created from the master branch. To conclude the release process, changes from the master branch, including all updates and hotfixes, are merged back into the develop branch.
 
+For more information on the quarterly release process, please refer to the [Release Process](./release_process.md) document.
+
 ### Hotfix Release
 If a critical bug or issue is found in the master branch, a hotfix branch should be created from the master branch. The hotfix branch should be used to fix the critical bug or issue. Once the hotfix branch is stable, it can be merged into the master branch and tagged with the release version. The master branch will then be merged back into the develop branch to ensure that the develop branch is up to date with the latest changes.
```

Comment:
Can we add some details on the standalone hotfix process to the release_process document?  ie. a hotfix that is done outside of a regular release.  For example to specify that all the version numbers in pom's, tags etc should also be updated for hotfix, and detail the process of creating the hofix from main and merging back into develop, etc.

Reply:
I've added a section to the `release_process.md` file for the standalone hotfix release process!

Reply:
Looks good!